### DISCUSSION TITLE
How does it compare to Traefik?

### DISCUSSION THREAD

**Discussion by lrkwz** [NginxProxyManager/nginx-proxy-manager#3242] [Q&A]
It would be very interesting a deep comparison between the two; at a first glance they seem to be quite overlapping.
(Results from the discussion may feed the FAQ document).

**Comment by shanelord01** [comment]
IMHO the comparison you are looking for is really NGINX vs Traefik.

NGINX Proxy Manager is an easy web UI to setup, control and manage NGINX and doesn't really have a comparison in Traefik or Caddy world.

Traefik only has a UI for monitoring - all config is done via config files (and auto discovery) but nothing via the UI.
